sbin/camcontrol/attrib.c

```diff
@@ -39,6 +39,7 @@
 #include <sys/types.h>
 #include <sys/endian.h>
 #include <sys/sbuf.h>
+#include <sys/param.h>
```

Comment:
As a general rule, <sys/param.h> should be listed first, and the first include should either be <sys/types.h> or <sys/param.h> (but not both, param.h includes types.h).  Thus, for this file (and several others here), you should replace the <sys/types.h> include with <sys/param.h>.